postgresql/manifests/postgres-with-backup/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base/upstream
```

Comment:
You are generating the files into `manifests/upstream` which is unused. You are using `manifests/base/upstream`, that is not updated on generation. Looks like it's wrong.

Reply:
thanks will fix in our repo